lib/mpool/src/mpool.c

```diff
@@ -224,12 +234,14 @@ mpool_create(const char *home, const struct mpool_cparams *cparams)
     for (i = MP_MED_BASE; i < MP_MED_COUNT; i++) {
         struct mclass_params mcp = {0};
 
-        /* If capacity path is the default, automatically create it */
-        if (i == MP_MED_CAPACITY) {
+        /* If the capacity/pmem path is the default, automatically create it */
+        if (i == MP_MED_CAPACITY || i == MP_MED_PMEM) {
             char   path[PATH_MAX];
             size_t n;
 
-            n = snprintf(path, sizeof(path), "%s/%s", home, MPOOL_CAPACITY_MCLASS_DEFAULT_PATH);
+            n = snprintf(path, sizeof(path), "%s/%s", home,
```

Comment:
Is home normalized here?

Reply:
I presume so, haven't changed anything in the logic on how/when hse_kvdb_create() calls mpool_create()...

Reply:
Just wanted to make sure we didn't fall into the double forward slash trap here.